Packs/GsuiteAuditor/Integrations/GsuiteAuditor/README.md

```diff
@@ -1,5 +1,12 @@
 G Suite Auditor is an integration that receives audit logs from G Suite's different applications - admin, drive, calendar, and more.
 
+## Required Permissions:
+In order to use the integration you will need:
+* Admin account.
+* Permission to the following scope - https://www.googleapis.com/auth/admin.reports.audit.readonly .
+
+For more information, see the integrations description.
```

Comment:
Usually, we are saying these exact words in the integration description (For more information, see the integrations documentation) as it makes more sense to have more information on the integration docs than description. What is there which is not here?

Reply:
It is the same information. As a client couldn't find the permission section I added this.